code/datums/diseases/tuberculosis.dm

```diff
@@ -3,11 +3,11 @@
 	name = "Fungal tuberculosis"
 	max_stages = 5
 	spread_text = "Airborne"
-	cure_text = "Spaceacillin & salbutamol"
-	cures = list("spaceacillin", "salbutamol")
+	cure_text = "Pan-Galactic Gargle Blaster"
+	cures = list("gargleblaster")
```

Comment:
you made the cure a bit harder but ok

Reply:
bartending is easy

Reply:
What are you smoking, gargleblaster is probably the most common drink and easy as hell to mix
Even the dispensers are easy to reach